python/src/trezorlib/toif.py

```diff
@@ -63,9 +63,9 @@ def _to_grayscale(data: bytes) -> bytes:
 
 @attr.s
 class Toif:
-    mode = attr.ib()  # type: firmware.ToifMode
-    size = attr.ib()  # type: Tuple[int, int]
-    data = attr.ib()  # type: bytes
+    mode: firmware.ToifMode = attr.ib()
```

Comment:
we could get rid of `= attr.ib()` by enabling `@attr.s(auto_attribs=True)`

Reply:
Let's do this in some other PR, please